app/main.py

```diff
@@ -1,25 +1,71 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_value: int, max_value: int) -> None:
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, obj: object, objtype: type = None) -> int:
+        return getattr(obj, self.protected_name)
+
+    def __set__(self, obj: object, value: Any) -> None:
+        if not isinstance(value, int):
+            raise ValueError("Value must be integer.")
+        elif not (self.min_value <= value <= self.max_value):
+            raise ValueError(
+                f"Value must be between {self.min_value} and {self.max_value}"
+            )
+        setattr(obj, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    age = IntegerRange(0, 150)
+    weight = IntegerRange(0, 500)
+    height = IntegerRange(0, 300)
```

Comment:
I don`t think we need this. It can work without this

Reply:
some really old guy will be really sad (

Reply:
Resolved